doc/api/tls.md

```diff
@@ -1154,7 +1154,7 @@ added: v0.11.13
 -->
 
 The default curve name to use for ECDH key agreement in a tls server. The
-default value is `'prime256v1'` (NIST P-256). Consult [RFC 4492] and
+default value is `'auto'`. Consult [RFC 4492] and
```

Comment:
I searched RFC 4492 and FIPS 186-4 for the word `auto`, and it does not appear. Could you provide a little more text that says something about what this new value actually does? IIRC, this curve is used for creation of ephemeral ECDH keys, is it that auto will cause the curve used to be the same as the curve in the server's EC certificate?